live-examples/js-examples/regexp/regexp-character-classes.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">var chessStory = "He played the King in a8 and she moved her Queen in c2.";
```

Comment:
We're using `const` and `let` in new examples now, and slowly updating existing ones.